File: app/src/main/java/com/example/lucas/bacias_hidrograficas/MapDescriptionSP.java
package com.example.lucas.bacias_hidrograficas;

import android.os.Bundle;
import android.webkit.WebView;
import android.widget.Button;

import androidx.appcompat.app.AppCompatActivity;
import androidx.appcompat.widget.Toolbar;
import androidx.core.view.GravityCompat;
import androidx.drawerlayout.widget.DrawerLayout;

public class MapDescriptionSP extends AppCompatActivity {

    WebView webView;
    Button button;

    @Override
    public void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.desc_map_sp);
        button = findViewById(R.id.b_close);
        button.setOnClickListener(view -> finish());

        Toolbar toolbar = findViewById(R.id.toolbar);
        setSupportActionBar(toolbar);
        setTitle("Descrição");

        webView = findViewById(R.id.webview2);
        String text = "<html><body align=\"justify\" style=\"font-size:15px\">"
                + getString(R.string.descricao_bacias_sp)
                + "</body></html>";
        webView.loadData(text, "text/html", "UTF-8");
    }

    @Override
    public void onBackPressed() {
        DrawerLayout drawer = findViewById(R.id.desc_map_sp);
        if (drawer.isDrawerOpen(GravityCompat.START)) {
            drawer.closeDrawer(GravityCompat.START);
        } else {
            super.onBackPressed();
        }
    }

}

File: app/src/main/java/com/example/lucas/bacias_hidrograficas/saoPauloMap/description/MapDescriptionSP.kt
package com.example.lucas.bacias_hidrograficas.saoPauloMap.description

import android.os.Bundle
import androidx.appcompat.app.AppCompatActivity
import com.example.lucas.bacias_hidrograficas.R
import com.example.lucas.bacias_hidrograficas.databinding.DescMapSpActivityBinding

class MapDescriptionSP : AppCompatActivity() {

    private val binding: DescMapSpActivityBinding by lazy {
        DescMapSpActivityBinding.inflate(layoutInflater)
    }

    public override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(binding.root)
        binding.close.bClose.setOnClickListener { finish() }

        setSupportActionBar(binding.content.toolbar)
        title = "Descrição"

        val text = ("<html><body align=\"justify\" style=\"font-size:15px\">"
                + getString(R.string.descricao_bacias_sp)
                + "</body></html>")
        binding.content.webview2.loadData(text, "text/html", "UTF-8")
    }
}
